middleware/HuaweiCloud-Kafka-autofill.yml

```diff
@@ -251,6 +251,57 @@ serviceConfigurationManage:
         isDataLost: true
         isServiceInterrupted: true
       isReadOnly: false
+serviceActions:
+  - name: upgrade
+    type: ansible
+    actionManageScripts:
+      - changeHandler: kafka-broker
+        runOnlyOnce: false
+        ansibleScriptConfig:
+          playbookName: "middleware/ansible/kafka/kafka-container-manage.yml"
+          virtualEnv: ""
+          pythonVersion: "3.10"
+          repoUrl: https://github.com/eclipse-xpanse/xpanse-samples
+          branch: main
+          requirementsFile: "requirements.txt"
+          galaxyFile: ""
+          ansibleInventoryRequired: false
+    actionParameters:
+      - name: kafka_cfg_message_max_bytes
```

Comment:
we need some other action parameters which are really valid for upgrade of a kafka broker.